indico/modules/events/models/reviews.py

```diff
@@ -84,44 +94,74 @@ def get_reviewer_render_data(self, user):
                 'reviews': reviews}
 
 
+class ProposalMixin(object):
```

Comment:
Docstrings here maybe as well? Class and `proposal_type`, `call_for_proposals_attr`, and `revisions_enabled`.